### PR TITLE
First task: adding myself as a member!

### PR DESCRIPTION
**Pull request by mohammedri**
This is a PR to add my picture to the blueprint site.

### CODE REVIEW

File: src/app/team-page/team-page.component.html [hunk 1]
```diff
@@ -117,6 +117,11 @@
         <p class="name">Nicholas Kam</p>
         <p class="title">project developer</p>
       </div>
+      <div class="col-md-3 col-sm-6">
```
Comment (chrisjluc) [inline src/app/team-page/team-page.component.html]: You added yourself in the alumni section!

**Comment by mohammedri** [comment]
@chrisjluc fixed.

File: src/app/team-page/team-page.component.html (hunk 1 above)
Comment (mohammedri) [inline src/app/team-page/team-page.component.html]: *facepalm*

File: src/app/team-page/team-page.component.html [hunk 2]
```diff
@@ -84,6 +84,14 @@
       </div>
     </div>
 
+    <div class="row">
+      <div class="col-md-3 col-sm-6">
+        <img src="https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fprofiles%2Fmohammed.jpg?alt=media&token=885a0597-327c-41a6-b52a-215fc9a2cec0">
+        <p class="name">Mohammed Ridwanul Islam</p>
+        <p class="title">Project Lead</p>
```
Comment (chrisjluc) [inline src/app/team-page/team-page.component.html]: lower case `project lead`

File: src/app/team-page/team-page.component.html [hunk 3]
```diff
@@ -117,6 +125,6 @@
         <p class="name">Nicholas Kam</p>
         <p class="title">project developer</p>
       </div>
-    </div>
```
Comment (chrisjluc) [inline src/app/team-page/team-page.component.html]: add space back

Comment (mohammedri) [inline src/app/team-page/team-page.component.html]: Fixed

File: src/app/team-page/team-page.component.html (hunk 2 above)
Comment (mohammedri) [inline src/app/team-page/team-page.component.html]: fixed

**Comment by chrisjluc** [comment]
You will need to add yourself to the row now, instead of creating two new rows.

**Comment by hkjayakumar** [comment]
The row is full, so we need to create a new row anyway 😛

**Comment by hkjayakumar** [comment]
LGTM 👍

**Pull request merged**